tutorials/katacoda/1-serving-intro/step2.md

```diff
@@ -0,0 +1,26 @@
+## Deploy and autoscale application
+
+We are going to deploy the [Hello world sample web application](https://knative.dev/docs/serving/samples/hello-world/helloworld-go/). This basic web application reads in an env variable TARGET and prints `Hello ${TARGET}!`. If TARGET is not specified, it will use `World` as the TARGET.
+
+We will now deploy the application by specifying the image location and the `TARGET` env variable.
+
+Knative defines a `service.serving.knative.dev` CRD to control the lifecycle of the application (not to be confused with kubernetes service). We will use the `kn` cli to create the Knative service:
```

Comment:
Do you want to mention something that this step will take up to a minute, at least when I try it is taking about 1 minute?
Is to avoid the user thinking that something is wrong.

Reply:
```
$ kn service create demo --image gcr.io/knative-samples/helloworld-go --env TARGET=blue
Creating service 'demo' in namespace 'default':

  0.155s The Route is still working to reflect the latest desired specification.
  0.218s Configuration "demo" is waiting for a Revision to become ready.
 50.677s ...
 50.816s Ingress has not yet been reconciled.
 50.874s Waiting for Envoys to receive Endpoints data.
 51.110s Waiting for load balancer to be ready
 51.329s Ready to serve.
 ```

Reply:
added a note.